registry/registry_db.go

```diff
@@ -348,6 +361,33 @@ func getNamespaceJwksByPrefix(prefix string, approvalRequired bool) (*jwk.Set, e
 	return &set, nil
 }
 
+func getNamespaceStatusById(id int) (RegistrationStatus, error) {
+	if id < 1 {
+		return "", errors.New("Invalid id. id must be a positive number")
```

Comment:
Are these IDs ever not integers? Just for clarity, we might replace `number` with `integer` if that's not the case.